### ISSUE TITLE
stepMergePullRefs in metapipeline.go needs to handle an empty base sha

### ISSUE THREAD

**Issue by hferentschik** [jenkins-x/jx#4604]
```go
func stepMergePullRefs(pullRefs prow.PullRefs) syntax.Step {
	args := []string{"--baseBranch", pullRefs.BaseBranch, "--baseSHA", pullRefs.BaseSha}
	for _, mergeSha := range pullRefs.ToMerge {
		args = append(args, "--sha", mergeSha)
	}

	step := syntax.Step{
		Name:      mergePullRefsStepName,
		Comment:   "Pipeline step merging pull refs",
		Command:   "jx step git merge",
		Arguments: args,
	}
	return step
}
```

The baseSha can be empty in which case `--baseSHA` should not be specifed.

**Comment by hferentschik** [comment]
Currently when the `baseSha` is empty, the build-step-merge-pull-refs step will error with:

```
Error: flag needs an argument: --baseSHA
```

**Comment by hferentschik** [comment]
fixed by pr #4608
